config/routes.rb

```diff
@@ -24,6 +24,11 @@
   resources :tools, only: %i[index]
   namespace :tools do
     resource :character_counter, only: %i[show create]
-    resource :stamina_calculator, only: %i[show create]
+    resource :stamina_calculator, only: %i[show create] do
+      collection do
+        get 'stamina', to: 'stamina_calculators#show_stamina'
+        post 'stamina', to: 'stamina_calculators#create_stamina'
```

Comment:
```suggestion
        get 'stamina_from_time', to: 'stamina_calculators#show_stamina'
        post 'stamina_from_time', to: 'stamina_calculators#create_stamina'
```
多少長くても、より分かりやすくした方が良い